framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -417,10 +418,8 @@ private static int compareTo(ColumnList list1, ColumnList list2,
         String string1 = (String) o1;
         String string2 = (String) o2;
         if (orderByColumns != null) {
-          String columnName;
           String fieldName;
-          String nextFieldName = "";
```

Comment:
Yeah, it's not used. I am not aware of any reason. I guess no reason would be valid if it's not used. Just checked with Robert (he is the author of this part of the code) that he might have left it there inadvertently.